metals-docs/src/main/scala/docs/GenericModifier.scala

```diff
@@ -10,18 +10,21 @@ class GenericModifier extends StringModifier {
 
   override def process(info: String, code: Input, reporter: Reporter): String =
     s"""
-       |## Gitignore `project/metals.sbt` `.metals/` and `.bloop/`
+       |## Gitignore `project/metals.sbt` `.metals/` `.ammonite/` and `.bloop/`
```

Comment:
```suggestion
       |## Files and Directories to include in your Gitignore
```
What do you think of instead of listing them all in the actual header, we just change it to be more generic here, and then be more descriptive down below?